codex-rs/mcp-server/src/message_processor.rs

```diff
@@ -538,6 +552,123 @@ impl MessageProcessor {
         });
     }
 
+    /// Send a message to a running Codex session using typed ConversationSendMessageArgs.
+    async fn handle_tool_call_send_message(
```

Comment:
Can we use this as a chance to break apart these large files. Maybe we make a subfolder for tool call handlers and then it takes `message_processor: &MessageProcessor` instead of `&self`